flink-formats/flink-avro-confluent-registry/src/main/java/org/apache/flink/formats/avro/registry/confluent/ConfluentRegistryAvroDeserializationSchema.java

```diff
@@ -84,14 +86,50 @@ private ConfluentRegistryAvroDeserializationSchema(
      */
     public static ConfluentRegistryAvroDeserializationSchema<GenericRecord> forGeneric(
             Schema schema, String url, int identityMapCapacity) {
+        return forGeneric(schema, url, identityMapCapacity, null);
+    }
+
+    /**
+     * Creates {@link ConfluentRegistryAvroDeserializationSchema} that produces {@link
+     * GenericRecord} using provided reader schema and looks up writer schema in Confluent Schema
+     * Registry.
+     *
+     * @param schema schema of produced records
+     * @param url URL of schema registry to connect
+     * @param registryConfigs map with additional schema registry configs (for example SSL
+     *     properties)
+     * @return deserialized record in form of {@link GenericRecord}
+     */
+    public static ConfluentRegistryAvroDeserializationSchema<GenericRecord> forGeneric(
+            Schema schema, String url, @Nullable Map<String, ?> registryConfigs) {
+        return forGeneric(schema, url, DEFAULT_IDENTITY_MAP_CAPACITY, registryConfigs);
+    }
+
+    /**
+     * Creates {@link ConfluentRegistryAvroDeserializationSchema} that produces {@link
+     * GenericRecord} using provided reader schema and looks up writer schema in Confluent Schema
+     * Registry.
+     *
+     * @param schema schema of produced records
+     * @param url URL of schema registry to connect
+     * @param identityMapCapacity maximum number of cached schema versions (default: 1000)
```

Comment:
Why is the default documented here? How would I pass an `int` that uses the default?